digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/dao/linearasset/PostGISSpeedLimitDaoSpec.scala

```diff
@@ -64,7 +63,7 @@ class PostGISSpeedLimitDaoSpec extends FunSuite with Matchers {
         RoadLink("080f5065-5ce7-4c58-b838-d0b4daf84adc:1", List(Point(0.0, 0.0), Point(370.0, 0.0)), 370.0, Municipality, 1, TrafficDirection.UnknownDirection, CableFerry, None, None, Map("MUNICIPALITYCODE" -> BigInt(235))),
         RoadLink("080f5065-5ce7-4c58-b838-d0b4daf84adc:1", List(Point(0.0, 0.0), Point(370.0, 0.0)), 370.0, Municipality, 1, TrafficDirection.UnknownDirection, UnknownLinkType, None, None, Map("MUNICIPALITYCODE" -> BigInt(235)))
       )
-      val dao = new PostGISSpeedLimitDao(MockitoSugar.mock[RoadLinkClient], MockitoSugar.mock[RoadLinkService])
+      val dao = new PostGISSpeedLimitDao(MockitoSugar.mock[RoadLinkService])
```

Comment:
Täälläkin ehkä pärjäis yhdellä daolla. Tai kahdella, jos tarvii olla osalla toi mock ja osalla null injektio.